### PR TITLE
fix(comments): Extend Vue before calling `new` on it

### LINKED ISSUE #46630: [Bug]: File comments broken
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The comments for files section is broken

```
TypeError: f is not a constructor
    mount comments-activity-tab.ts:24
    NextJS 26
    mounted SidebarTab.vue:104
    VueJS 16
index-BK2GMarY.mjs:2:23954
    NextJS 29
    mounted SidebarTab.vue:104
    VueJS 16
```

### Steps to reproduce

1. Login to Nextcloud
2. Open files app
3. Open sidebar
4. Change to activity tab
5. The field to write a comment is missing

[Screencast from 2024-07-19 11-57-04.webm](https://github.com/user-attachments/assets/a7958d3b-a3c3-4f3e-a75a-1227629d53f6)

### Expected behavior

Able to leave a comment ;)

### Installation method

None

### Nextcloud Server version

master

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

_No response_

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

Firefox Developer Edition 128.0b9

### PR DESCRIPTION
**Pull request by susnux**
* Resolves: https://github.com/nextcloud/server/issues/46630

## Summary

https://github.com/nextcloud/server/pull/45848 introduced a regression as the Vue was not extended but the Vue component was directly tried to instantiate which does not work as a Vue component is basically an Object that does not has any constructor.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by kesselb — APPROVED** [review]
Thanks, it does work again ;)

**Review by Pytal — APPROVED** [review]

File: apps/comments/src/comments-activity-tab.ts [hunk 1]
```diff
@@ -24,17 +24,18 @@ export function registerCommentsPlugins() {
 
 			if (!ActivityTabPluginView) {
 				const { default: ActivityCommmentAction } = await import('./views/ActivityCommentAction.vue')
-				ActivityTabPluginView = ActivityCommmentAction
+				/** @ts-expect-error Types are broken for Vue2 */
+				ActivityTabPluginView = Vue.extend(ActivityCommmentAction)
```
Comment (artonge) [inline apps/comments/src/comments-activity-tab.ts:28]: :pray: 
```suggestion
				ActivityTabPluginView = Vue.extend(ActivityCommentAction)
```

Comment (artonge) [reply apps/comments/src/comments-activity-tab.ts]: Done

**Review by artonge — APPROVED** [review]

**Pull request merged**